### DISCUSSION TITLE
Loki returns upstream connect error or disconnect/reset before headers when querying a large volume of logs

### DISCUSSION THREAD

**Discussion by KevinCosta8** [grafana/grafana#62931] [Observability]
I'm running Loki Simple Scalable with three readers, three writers, and an nginx gateway. I have logs and indexes going to an s3 bucket. When querying two days of data, the volume loads fine, but it does take a few seconds to come in. When I query 7 days of logs, I get that error. Is there a flag in the gateway config I missed?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
